ext/vscode/package.nls.json

```diff
@@ -1,7 +1,7 @@
 {
     "azure-dev.commands_category": "Azure Developer CLI (azd)",
 
-    "azure-dev.commands.cli.init.title": "Initialize App (init)",
+    "azure-dev.commands.cli.init.title": "Generate Azure Deployment Script (init)",
```

Comment:
From azd's perspective, maybe something like "Intialize App for Azure"? or "App" is not quite the right word here for Java context, how about "Initialize for Azure"?

Reply:
@taoxu0903, what's your opinion?

Reply:
@saragluna LGTM, "Initialize for Azure" is more consistent naming with other names in AZD extension.